tests/test_rebase_consensus.py

```
@@ -0,0 +1,864 @@
+        assert result.success is True
+        mock_providers.assert_not_called()
+
+    @pytest.mark.asyncio
+    @patch("sova.git.rebase._create_providers")
+    @patch("sova.git.rebase._load_consensus_config")
+    @patch("sova.git.rebase._configure_diff3", new_callable=AsyncMock)
+    @patch("sova.git.rebase.run", new_callable=AsyncMock)
+    async def test_consensus_disabled_when_providers_unavailable(
+        self,
+        mock_run: AsyncMock,
+        _mock_diff3: AsyncMock,
```

Ruff (0.16.1)

[warning] Fixture `_mock_diff3` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)